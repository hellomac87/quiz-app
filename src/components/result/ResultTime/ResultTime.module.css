.container {
    @apply flex items-center justify-between mb-4;
}
