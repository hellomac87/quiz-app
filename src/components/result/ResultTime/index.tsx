import Card from 'src/components/common/Card';

import styles from './ResultTime.module.css';

type Props = {
    time: string;
};

function ResultTime({ time }: Props) {
    return (
        <Card className={styles.container}>
            <span>{'문제 푸는데 걸린 시간'}</span>
            <span>{time}</span>
        </Card>
    );
}

export default ResultTime;
