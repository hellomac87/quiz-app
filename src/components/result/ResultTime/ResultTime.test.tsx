import renderer from 'react-test-renderer';

import ResultTime from './index';

describe('ResultTime test', () => {
    test('snapshot: no Props', () => {
        const tree = renderer.create(<ResultTime time={'time'} />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
