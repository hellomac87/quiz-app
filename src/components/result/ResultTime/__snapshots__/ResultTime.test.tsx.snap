// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultTime test snapshot: no Props 1`] = `
<div
  className="container container"
>
  <span>
    문제 푸는데 걸린 시간
  </span>
  <span>
    time
  </span>
</div>
`;
